deploy/olm-catalog/opendatahub/1.2.0/manifests/opendatahub-operator.v1.2.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,361 @@
+                    "path": "odh-dashboard"
+                  }
+                },
+                "name": "odh-dashboard"
+              }
+            ],
+            "repos": [
+              {
+                "name": "kf-manifests",
+                "uri": "https://github.com/opendatahub-io/manifests/tarball/v1.4.0-rc.2-openshift"
+              },
+              {
+                "name": "manifests",
+                "uri": "https://github.com/opendatahub-io/odh-manifests/tarball/v1.2.0"
+              }
+            ]
+          }
+        }
+      ]
+    capabilities: Deep Insights
+    categories: AI/Machine Learning, Big Data
+    description: "Open Data Hub Operator for deployment and management of Open Data Hub and Kubeflow on OpenShift"
+    support: Open Data Hub
+    repository: https://github.com/opendatahub-io/opendatahub-operator
+    createdAt: '2021-01-31T00:00:00Z'
+    containerImage: quay.io/opendatahub/opendatahub-operator:v1.2.0
+    certified: 'False'
+  name: opendatahub-operator.v1.2.0
+  namespace: placeholder
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions:
+    owned:
+      - description: KfDef is the Schema for the applications API
+        kind: KfDef
+        name: kfdefs.kfdef.apps.kubeflow.org
+        version: v1
+        displayName: Open Data Hub
+        group: kfdef.apps.kubeflow.org
+  description: |
+    The Open Data Hub is a machine-learning-as-a-service platform built on Red Hat's Kubernetes-based OpenShift® Container Platform. Open Data Hub integrates multiple AI/ML open source components into one operator that can easily be downloaded and installed by OpenShift users.
+
+    Open Data Hub operator allows users to install and manage Open Data Hub and Kubeflow components. Users can mix and match tools from each project to fulfill the needs of their use case. Each of the
+    projects share some components, but can be mostly seen as an extension of each other to provide a complete solution for both novice and skilled enterprise users.
+
+    ### Components
+    * JupyterHub v0.3.5 - Open source multi-user JupyterLab notebook platform w/ GPU support and **NEW** High Availability.
+    * Trino v355 - Distributed analytics SQL database that supports multiple data sources
+    * Hue v4.8.0 - A service that provides data exploration on Hive and S3 buckets.
+    * Spark Thrift Server - A service that allows JDBC clients run Spark SQL queries.
+    * Open Data Hub Dashboard v1.0  - A web dashboard that displays installed Open Data Hub components with easy access to component UIs and documentation
+    * Elyra v2.2.4 - JupyterLab notebooks with support for AI workflows
+    * Ceph Nano v0.7 - Minimal Object Storage provided by Ceph for development use
+    * Apache Spark v2.4.5 - Unified analytics engine for large-scale data processing
+    * Prometheus v2.16.0 - Monitoring and alerting tool
+    * Grafana v7.1.1 - Data visualization and monitoring
```

Comment:
We might need to update Grafana, seeing errors while deploying on 4.10 https://github.com/opendatahub-io/opendatahub-operator/issues/155

Reply:
Resolving this since you closed #155.  Since we removed the `startingCSV` from the Grafana operator subscription in odh-manifests we are reliant on OperatorHub providing a working operator version